site/content/en/latest/tasks/observability/proxy-trace.md

```diff
@@ -347,5 +347,17 @@ spec:
 EOF
 ```
 
+### Tempo and appProtocol
 
-[envoy-proxy-crd]: ../../api/extension_types#envoyproxy
\ No newline at end of file
+When using Grafana Tempo it is important to note that the [tempo-distributed Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/tempo-distributed)
+will omit a `appProtocol` in the `tempo-distributor` Service. Without this protocol hint, Envoy will send traces to Tempo using the wrong protocol: HTTP, instead of GRPC.
+
+You can add the protocol hint to the `tempo-distributor` Service, by supplying these values to the "tempo-distributed" Helm chart:
+
+```yaml
+distributor:
+  appProtocol:
+    grpc: grpc
```

Comment:
instead of `grpc` in the key, can we use `kubernetes.io/h2c` instead ?
this is what upstream would ideally like
https://gateway-api.sigs.k8s.io/guides/backend-protocol/